src/main/proto/wfa/measurement/api/v2alpha/exchange_step.proto

```diff
@@ -55,4 +65,37 @@ message ExchangeStep {
     // The step has permanently failed. Terminal state.
     FAILED = 4;
   }
+
+  // Represents how to access an output/input.
+  message SharedData {
+    // Encrypted, serialized `SignedData` messages containing serialized
+    // `Location` payloads.
+    bytes encrypted_signed_location = 1;
+
+    message Location {
+      // This should correspond to one of the values in the
+      // `ExchangeWorkflow.Step.output_labels` map for a step.
+      string label = 1;
+
+      oneof location {
+        // Represents a file to download over HTTPS. Since SharedOutput is only
+        // shared via an encrypted SignedData message, it is safe to put URLs
+        // with access tokens in here.
+        string https_uri = 2;
+
+        // Represents a query to execute under Google Cloud's BigQuery.
+        string google_big_query = 3;
+
+        // Represents a file accessible via Google Cloud Storage. While
+        // sometime HTTPS can be used directly, some may prefer to use ACLs
+        // rather than access tokens. This should be a standard GCS path of the
+        // format "gcs://bucket-name/and/some/path".
+        string google_cloud_storage = 4;
+
+        // Represents a file accessible via AWS S3. While sometimes HTTPS can be
+        // used directly, some may prefer to use ACLs rather than access tokens.
+        string amazon_s3 = 5;
```

Comment:
if we can put in the comment an example, e.g., "s3://bucket-name/and/some/path", that would be helpful

Reply:
Done.